src/CredentialRepository.php

```diff
@@ -215,7 +215,7 @@ public function __unserialize(array $data): void
      * The __serialize() magic method will be automatically used instead of this
      *
      * @return string
-     * @deprecated will be removed in 5.0
+     * @deprecated 4.5.0 Use __serialize() insteaed
```

Comment:
No corresponding deprecation notice?
```suggestion
     * @deprecated 4.5.0 Use __serialize() instead
```

Reply:
Updated